AddSevenPS/LibraryTest.fs

```diff
@@ -0,0 +1,10 @@
+﻿module AddSevenPS.LibraryTest
+
+open Xunit
+open AddSevenPS
+
+[<Fact>]
+let `` AddSeven.addSeven -> " 1 => 8" `` () =
+    let result =  AddSeven.addSeven(1)
+    let expected = 8
+    Assert.Equal(expected, result)

```

Comment:
This test used the XUnit based setup 

Some relevant blogs
- https://fsprojects.github.io/FsUnit/xUnit.html
- https://khalidabuhakmeh.com/write-unit-tests-with-fsharp-and-xunit